src/clients/python/examples/ipc.py

```diff
@@ -1,10 +1,16 @@
-import sys
+# /// script
+# requires-python = ">=3.13"
+# dependencies = [
+#     "webview-python",
+# ]
+#
+# [tool.uv.sources]
+# webview-python = { path = "../" }
+# ///
```

Comment:
This is [PEP 723](https://peps.python.org/pep-0723/) inline metadata, which allows you to specify the dependencies for a standalone script _inline_ as a comment. 

I added this with:

```sh
cd src/client/python
uv add --script examples/ipc.py .
```

uv will now parse and run the script an isolated environment with the required dependencies.